src/service/fixed-layer.js

```diff
@@ -128,6 +128,32 @@ export class FixedLayer {
   }
 
   /**
+   * Apply or reset transform style to fixed elements
+   * @param {?string} transform
+   */
+  transformMutate(transform) {
+    if (transform) {
+      // Apply transform style to all fixed elements
+      this.fixedElements_.forEach(e => {
+        if (e.fixedNow && e.top) {
+          if (e.transform && e.transform != 'none') {
+            setStyle(e.element, 'transform', e.transform + ' ' + transform);
+          } else {
+            setStyle(e.element, 'transform', transform);
+          }
+        }
+      });
+    } else {
+      // Reset transform style to all fixed elements
+      this.fixedElements_.forEach(e => {
+        if (e.fixedNow && e.top) {
+          setStyle(e.element, 'transform', '');
```

Comment:
We are setting it above only temporarily for the duration of transition (e.g. 200ms) and then reset it completely. So, we'd generally like to reduce the scope/time where our customizations are applied.

Consider this case:

```
.el {
  transform: scale(0.5);
}

@media (min-width: 500px) {
  .el {
    transform: scale(1.5);
  }
}
```

We start in portrait mode with viewport width of 400px.

And we manipulate header:
```
<div class="el" style="transform:scale(0.5)  translateY(-20px)">
```

After transition done, instead of clearing transform out (as we do here), we'll restore the old value:
```
<div class="el" style="transform:scale(0.5)">
```

Everything is great so far.

But now we'll switch to landscape mode and viewport width becomes `600px`, which triggers `transform: scale(1.5);` rule in media query. However, it's ignored since we still have `transform: scale(0.5)` on the element.

We can, of course, reset `transform` to the new calculated value on each resize. But my point here is why do that if we can simply reset it and "give it back" to the stylesheet.